src/hooks/useCourtContracts.js

```diff
@@ -351,6 +351,32 @@ export function useAppealFeeAllowance(owner) {
   return allowance
 }
 
+export function useActiveBalanceOfAt(juror, termId) {
+  const jurorRegistryContract = useCourtContract(
+    CourtModuleType.JurorsRegistry,
+    jurorRegistryAbi
+  )
+  const [activeBalance, setActiveBalance] = useState(bigNum(-1))
+
+  useEffect(() => {
+    const getActiveBalanceOfAt = async () => {
+      if (!jurorRegistryContract) return
+
+      retryMax(() => jurorRegistryContract.activeBalanceOfAt(juror, termId))
+        .then(balance => {
+          setActiveBalance(balance)
+        })
+        .catch(err => {
```

Comment:
I'm not sure this will work as intended; `retryMax()` only retries if the inner function rejects with an error.

Here, we catch any errors in the promise and log it without re-throwing it.

Reply:
Well, yes this was originally the intention. What do you think we should do when the max retry attempts has been reached? 

Should we re throw the error and let it be handled by the global error?

Reply:
Maybe we can set an error state in this hook, so the component outside of it can choose to respond?